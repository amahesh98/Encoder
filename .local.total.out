CS251: . Ashwin Wed Feb 7 23:08:42 EST 2018  Total:  6 of 10
CS251: . Ashwin Sun Feb 11 23:37:51 EST 2018  Total:  6 of 100
CS251: . Ashwin Sun Feb 11 23:38:20 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 00:56:44 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 17:12:36 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 17:36:45 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 17:44:19 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 17:46:07 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 17:47:41 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 20:57:39 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 22:29:38 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 22:56:03 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 12 23:12:40 EST 2018  Total:  6 of 100
CS251: . Ashwin Tue Feb 13 00:18:35 EST 2018  Total:  6 of 10
CS251: . Ashwin Mon Feb 19 14:14:47 EST 2018  Total:  0 of 10
CS251: . Ashwin Mon Feb 19 18:54:42 EST 2018  Total:  0 of 0
CS251: . Ashwin Mon Feb 19 18:54:47 EST 2018  Total:  0 of 10
