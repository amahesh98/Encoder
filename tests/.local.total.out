CS251: . elkindi Sat Feb 3 14:34:17 EST 2018  Total:  140 of 140
CS251: . elkindi Sat Feb 3 14:37:44 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:37:50 EST 2018  Total:  8 of 8
CS251: . elkindi Sat Feb 3 14:39:37 EST 2018  Total:  8 of 8
CS251: . elkindi Sat Feb 3 14:41:21 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:42:17 EST 2018  Total:  80 of 80
CS251: . elkindi Sat Feb 3 14:42:25 EST 2018  Total:  100 of 100
CS251: . elkindi Sat Feb 3 14:58:29 EST 2018  Total:  0 of 88
CS251: . elkindi Sat Feb 3 14:59:29 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:59:34 EST 2018  Total:  80 of 88
CS251: . elkindi Sat Feb 3 14:59:50 EST 2018  Total:  80 of 88
CS251: . elkindi Sat Feb 3 15:00:57 EST 2018  Total:  88 of 88
CS251: . elkindi Sat Feb 3 15:02:53 EST 2018  Total:  112 of 112
CS251: . elkindi Sat Feb 3 15:05:14 EST 2018  Total:  80 of 80
CS251: . elkindi Sat Feb 3 15:05:33 EST 2018  Total:  100 of 100
CS251: . elkindi Sat Feb 3 15:12:14 EST 2018  Total:  2 of 10
CS251: . elkindi Sat Feb 3 15:15:46 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:16:11 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 15:17:42 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:18:05 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:20:07 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:25:12 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:31:16 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:32:21 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:33:13 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:33:59 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:43:43 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:43:48 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:45:15 EST 2018  Total:  0 of 80
CS251: . elkindi Sat Feb 3 15:48:47 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:48:49 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:48:52 EST 2018  Total:  0 of 80
CS251: . Ashwin Tue Feb 13 00:19:37 EST 2018  Total:  6 of 10
CS251: . Ashwin Tue Feb 13 01:09:06 EST 2018  Total:  6 of 10
CS251: . Ashwin Tue Feb 13 01:19:31 EST 2018  Total:  6 of 10
CS251: . Ashwin Wed Feb 14 22:53:17 EST 2018  Total:  6 of 10
CS251: . mahesh2 Wed Feb 21 01:06:40 EST 2018  Total:  8 of 100
CS251: . Ashwin Wed Feb 21 15:46:21 EST 2018  Total:  0 of 10
CS251: . Ashwin Mon Feb 26 02:43:43 EST 2018  Total:  6 of 100
CS251: . Ashwin Mon Feb 26 02:44:04 EST 2018  Total:  6 of 100
CS251: . Ashwin Mon Feb 26 02:44:33 EST 2018  Total:  0 of 80
